gnosis/safe-react
# fix: GTM-GA integration

## What it solves
Integration with GA

## How this PR fixes it
All tracking now sends "top-level" values in payloads and GA cookie removal on unloud was reintroduced.

## How to test it
Ensure that all tracking events are visible in GA when:
- Navigating through load/create Safe steppers.
- Clicking Safe 'overview' buttons.

When removing permissions for "Analytics" cookies, they should be removed.
